builder/build.go

```diff
@@ -161,3 +160,10 @@ func buildFlagString(nocache bool, squash bool, httpProxy string, httpsProxy str
 
 	return buildFlags
 }
+
+func debugPrint(message string) {
+
+	if val, exists := os.LookupEnv("debug"); exists && (val == "1" || val == "true") {
```

Comment:
Could add a `--debug` flag for turning this on across all commands, something for a seperate PR perhaps.

Reply:
Yes, that was kind of the theory behind pulling it out.  Suggest a separate PR since this one is a 'bug' fix